department-of-veterans-affairs/caseflow
# Reassign Correspondence Allows Reassignment over 60 Limit

Resolves [APPEALS-43250](https://jira.devops.va.gov/browse/APPEALS-43250)

# Description
When a supervisor assigns cases to a user, the supervisor can assign over 60 cases to the user. 

## Acceptance Criteria
- [x]  The workflow does not allow over 60 cases to be assigned to the user

## Testing Plan

1. sign in as MAIL_TEAM_SUPERVISOR_MAIL_INTAKE_USER
2. Navigate to correspondence
3. Click the assigned tab
4. In the rails console, validate how many tasks are assigned to MCWILJVACO by running CorrespondenceTask.where(assigned_to: User.find_by(css_id: 'MCWILJVACO')).count (expected to be 0, but can be more).
5. Select 58 cases that aren't assign to MCWILJVACO in the assigned tab.
6. Selected MCWILJVACO from the dropdown and click reassign.
7. A green success banner should appear showing that the 58 cases were assigned
8. Select 5 more cases that aren't assigned to MCWILJVACO.
9. Select MCWILJVACO from the dropdown and click reassign.
10. A green success banner appears showing the cases were assigned to MCWILJVACO.
11. Validate in the rails console by running CorrespondenceTask.where(assigned_to: User.find_by(css_id: 'MCWILJVACO')).count to see that the count is over 60.
